### PR TITLE
WRQ-1308: Update audio guidance of Dropdown component

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update audio guidance of Dropdown component
- When dropdown list is opened, read out `Dropdown list opened`
- When dropdown button is focused, read out `Dropdown` after the contents

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated aria-label in Dropdown
Updated DropdownList to use aria-labelledby to read only when the focus is entered into the list

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-1308

### CODE REVIEW

File: Dropdown/Dropdown.js [hunk 1]
```diff
@@ -337,7 +337,7 @@ const DropdownBase = kind({
 			<div {...calcAriaProps} {...rest}>
 				{title}
 				<DropdownButton
-					aria-label={ariaLabel}
+					aria-label={ariaLabel || placeholder + ` ${$L('Dropdown')}`}
```
Comment (seunghoh) [inline Dropdown/Dropdown.js:340]: You should read "Dropdown Button" when the user sets `aria-label`.

File: Dropdown/DropdownList.js [hunk 2]
```diff
@@ -131,24 +141,27 @@ const DropdownListBase = kind({
...
-				cbScrollTo={scrollTo}
-				dataSize={dataSize}
-				itemSize={ri.scale(itemSize)}
-				style={{
-					height: ri.scaleToRem((itemSize * dataSize) + 36),
-					width: typeof width === 'number' ? ri.scaleToRem(width) : null
-				}}
-			/>
+			<div role="region" aria-labelledby={`${id}_dropdownlist`}>
+				<div id={`${id}_dropdownlist`} aria-label={$L('Dropdown list opened')} />
```
Comment (seunghoh) [inline Dropdown/DropdownList.js:153]: Good idea!

**Review by seunghoh — CHANGES_REQUESTED** [review]

File: Dropdown/Dropdown.js (hunk 1 above)
Comment (juwonjeong) [reply Dropdown/Dropdown.js]: It's fixed to read "Dropdown Button" even when the user sets `aria-label`.

**Review by seunghoh — APPROVED** [review]
LGTM but Let's make a separate ticket for analyzing why it can't read the title when set `aria-label`.

**Pull request merged**